pipelineutilities/build/lib/pipelineutilities/validate_json.py

```diff
@@ -0,0 +1,248 @@
+# validate_json.py
+""" Common code to retrieve the nd_json_schema and to perform basic json validation against any json schema"""
+
+from jsonschema import validate, ValidationError
+from sentry_sdk import capture_exception
+
+
+def validate_json(json_to_test, schema_to_use, print_error_message=False):
+    """ Validate json against a schema """
+    results = False
+    try:
+        validate(instance=json_to_test, schema=schema_to_use)
+        results = True
+    except ValidationError as e:
+        results = False
+        if print_error_message:
+            print("Validate_json failed with this error message: ", str(e))
+        capture_exception(e)
+    return results
+
+
+nd_json_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Schema for validating ND.json",
+    "title": "Tester for ND.json schema",
+    "id": "nd.schema.json",
+    "type": "object",
+    "properties": {
+        "id": {"type": "string"},
```

Comment:
Please add descriptions to the properties. I believe the schema is something we'll want to publish and share (as its a gatekeeper for moving data through the pipeline).